src/octree/tests.rs

```diff
@@ -1,30 +1,32 @@
 #[cfg(test)]
 mod octree_serialization_tests {
+    use crate::octree::types::Albedo;
     use crate::octree::Octree;
     use crate::octree::V3c;
 
     #[test]
     fn test_octree_file_io() {
-        let mut tree = Octree::<u32>::new(4).ok().unwrap();
+        let red: Albedo = 0xFF0000FF.into();
```

Comment:
@davids91  

This is mostly a demonstration that we do not lose the "use u32 as hex-color directly" ability with the current `Albedo` implementation.

Reply:
Up until now I did not realize that this is more like AGRB code, rater than RGA '^^  Would it be much effort to reverse the byte order? 

Otherwise this looks really good!

Reply:
Na, we can do it of course... but you wrote "Just a sidenote: This implementation is actually changing the Byte order which is currently implemented. That is fine though because I would prefer the new order, but the tests are using opposite byte orders." further down.

Which do you prefer? I mean, with the current (outside this PR) byteorder, we could just write "FF0000" and let it be "FF0000FF" so "red with opacity 1" (or "red with alpha 1") while with "my" byteorder we cannot.

So I think yours is more elegant

Reply:
I'd like it to be as close to RGB codes as possible, so to the user/dev RGBA --> 0xRRGGBBAA

Reply:
Okay, so the current impl is already what you want then, right? No need for me to change anything?!

Reply:
Yes! :)